Files/UserControls/FileIcon.xaml

```diff
@@ -45,7 +45,7 @@
             HorizontalAlignment="Center"
             VerticalAlignment="Stretch"
             x:Load="{x:Bind ViewModel.LoadCustomIcon, Mode=OneWay}"
-            Source="{x:Bind CustomIconImageSource, Mode=OneWay}" />
```

Comment:
Isn't OneWay required so that it updates the icon source?

Reply:
This source is never updated and is never notified by OnPropertyChanged. OneWay is therefore not used.